man/systemd.network.xml

```diff
@@ -1207,6 +1207,18 @@ DuplicateAddressDetection=none</programlisting></para>
           carrier. Defaults to false. If enabled, and the <varname>IgnoreCarrierLoss=</varname> setting
           is not explicitly set, then it is enabled as well.</para>
 
+          <para>With this enabled, to make the interface enter the <literal>configured</literal> state,
+          which is important for <command>systemd-networkd-wait-online</command> working for the interface,
```

Comment:
"which is required to make <command>systemd-networkd-wait-online</command> work properly for the interface"